crates/common_enums/src/enums.rs

```diff
@@ -1561,3 +1561,25 @@ pub enum PayoutEntityType {
     Business,
     Personal,
 }
+
+#[derive(
+    Clone,
+    Copy,
+    Default,
+    Debug,
+    Eq,
+    Hash,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    strum::Display,
+    strum::EnumString,
+    ToSchema,
+)]
+#[router_derive::diesel_enum(storage_type = "pg_enum")]
```

Comment:
this is required only if the enum is stored in database, if not required then can be removed

Reply:
removed